mep/books/admin.py

```diff
@@ -353,11 +397,123 @@ class FormatAdmin(admin.ModelAdmin):
     fields = ("name", "uri", "notes")
 
 
-admin.site.register(Work, WorkAdmin)
+class GenreAdmin(admin.ModelAdmin):
+    list_display = ("name",)
+    # override default order to put notes last
+    fields = ("name",)
+    search_fields = ("name",)
+
+
+class WorkResource(ImportExportModelResource):
```

Comment:
is it normal/recommended to put all the resource code in admin, or would it make sense to put it somewhere else? does django-i-e have a recommended structure?

Reply:
I believe so? The docs aren't clear on this but I've usually seen it all in a single file. It's only accessible/usable from the admin interface so it seems reasonable to keep it there rather than put it into models.py, but up to you.